ads/google/a4a/docs/RTB-Implementation.md

```diff
@@ -0,0 +1,65 @@
+# AMP Ad Implementation Guide for RTB Ad Exchanges
+---
+## Objective
+ 
+This guide is designed to provide additional information for SSPs and Ad Exchanges that want to support AMP Ads in a Real-Time Bidding (RTB) environment.  The IAB's OpenRTB 2.5 spec is [here](http://www.iab.com/wp-content/uploads/2016/03/OpenRTB-API-Specification-Version-2-5-FINAL.pdf&sa=D&ust=1496436943725000&usg=AFQjCNFlig47imweXbZx-yuvW8LmlonzFw).
+ 
+For AMP Ads on AMP Pages, a major benefit is that if the ad response is AMP, then the ad can be rendered early by splicing it into the surrounding AMP page, with no delays and without risk to the page's UX.  The AMP page needs to protect itself from non-AMP content, so must delay the rendering of any ad that is not in AMP Format.
+ 
+# Background Docs
+* [AMP Ads for AMP Pages (Github)](https://github.com/ampproject/amphtml/issues/3133&sa=D&ust=1496436943729000&usg=AFQjCNFdEAhcweynv_C57XPDvquNmA-d_w)
+* [AMP Ad Creative Format Spec (Github)](https://github.com/google/amphtml/blob/master/extensions/amp-a4a/amp-a4a-format.md&sa=D&ust=1496436943730000&usg=AFQjCNFmQDH1dV9M0qmOXazSaWBozivMkQ)
+* [AMP Ads Overview (Github)](https://github.com/ampproject/amphtml/blob/master/ads/google/a4a/docs/a4a-readme.md&sa=D&ust=1496436943731000&usg=AFQjCNEs6s6zDQHzfSLw-W2x9oOF7EfU5w)
+* [AMP Ads Website from the AMP Project](http://www.ampproject.org/ads&sa=D&ust=1496436943732000&usg=AFQjCNEwQwnBRfEKRN6gftjeoPa_OYrlkQ)
+* [Example AMP Ads](https://ampbyexample.com/amp-ads/%23amp-ads/introduction&sa=D&ust=1496436943733000&usg=AFQjCNFwdvQ4G4V0TeBjKd7vfd1xohQQOQ)
+* [Speed comparison](https://ampbyexample.com/amp-ads/introduction/amp_ads_vs_non-amp_ads/&sa=D&ust=1496436943734000&usg=AFQjCNG4RdQVhjlh7Sbk9d5ylDptM4-Ffw): see how fast an AMP Ad loads in comparison to a regular ad. Best viewed on a 3G connection.
+* [Discussion in OpenRTB Dev Forum](https://groups.google.com/forum/%23!topic/openrtb-dev/0wyPsF5D07Q&sa=D&ust=1496436943736000&usg=AFQjCNFlwyrmUfG8v4VLgAuZUqSCroDAZw): RTB Specific Proposal
+ 
+## AMP Ads in RTB: High-Level Approach
+ 
+### RTB Bid Request
+ 
+Exchanges will need to indicate in the RTB bid request whether a page is built in AMP HTML, and any specific requirements or treatment of AMP Ads.  As of [OpenRTB 2.5](https://www.google.com/url?q=http://www.iab.com/wp-content/uploads/2016/03/OpenRTB-API-Specification-Version-2-5-FINAL.pdf&sa=D&ust=1496436943740000&usg=AFQjCNGuz1b-E6kOzHQr9RJ6Xb_DNAy_xQ), this is not yet included in the spec, but the proposed implementation to the IAB committee is as follows.
+ 
+**Site Object Additional field: amp**
+A new field is added to the 'Site' object of the OpenRTB standard to indicate whether a webpage is built on AMP HTML.  In OpenRTB 2.5, this is section 3.2.13.
+ 
+![RTB1](./img1.png)
```

Comment:
Do these tables have to be images?  It's not great for accessibility readers.  Consider making all RTB1,2,3... as inline tables.